src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/RedHatSeriesMenuCard/RedHatSeriesMenuCard.scss

```diff
@@ -2,13 +2,15 @@
   &__toggle-container {
     margin: var(--pf-global--spacer--lg) 0 0;
     border: 2px solid var(--pf-global--BorderColor--300);
+    flex-basis: 19%;
```

Comment:
?

Reply:
I want 5 flex items in a row with some space between them